src/assets/documents/api/button.html

```diff
@@ -14,11 +14,59 @@ <h2 class="docs-api-h2">
   Module: <code>MdButtonModule</code>
 </h2><h2>Directives</h2>
   
-    <h3 class="docs-api-h3 docs-api-class-name">MdButton</h3>
-<p class="docs-api-class-description">Material design button.
+    <h3 class="docs-api-h3 docs-api-class-name">MdButtonCssMatStyler</h3>
+<p class="docs-api-class-description">Directive whose purpose is to add the mat- CSS styling to this selector.
+</p>
+
+
+
+
+
+
+  
+    <h3 class="docs-api-h3 docs-api-class-name">MdRaisedButtonCssMatStyler</h3>
+<p class="docs-api-class-description">Directive whose purpose is to add the mat- CSS styling to this selector.
+</p>
+
+
+
+
+
+
+  
+    <h3 class="docs-api-h3 docs-api-class-name">MdIconButtonCssMatStyler</h3>
+<p class="docs-api-class-description">Directive whose purpose is to add the mat- CSS styling to this selector.
+</p>
+
+
+
+
+
+
+  
+    <h3 class="docs-api-h3 docs-api-class-name">MdFabCssMatStyler</h3>
+<p class="docs-api-class-description">Directive whose purpose is to add the mat- CSS styling to this selector.
+</p>
+
+
+
+
+
+
+  
+    <h3 class="docs-api-h3 docs-api-class-name">MdMiniFabCssMatStyler</h3>
+<p class="docs-api-class-description">Directive whose purpose is to add the mat- CSS styling to this selector.
```

Comment:
These style directives probably shouldn't be documented. Remove here and we will add PR to main repo to add docs-private notation